sktime/split/singlewindow.py

```diff
@@ -34,22 +34,49 @@
 class SingleWindowSplitter(BaseSplitter):
     r"""Single window splitter.
 
-    Split time series once into a training and test set.
-    See more details on what to expect from this splitter in :class:`BaseSplitter`.
+    Split time series into a single training and test set window.
 
-    Test window is defined by forecasting horizons
-    relative to the end of the training window.
-    It will contain as many indices
+    The training set is defined based on a "window".
+    The endpoint of the training set is determined by
+    `the size of the time series - fh[-1] - 1`,
+    and the starting point is calculated as `endpoint - window_length + 1`.
+    If the starting point is negative, it will be set to 0.
+
+    If the time points in the data are :math:`(t_1, t_2, \ldots, t_N)`, the training
+    windows will be all indices in the intervals
+
+    .. math:: [t_{N-fh[-1] - w}, \ldots, t_{N-fh[-1] - 1}]
```

Comment:
is this correct? That is, is the splitter `iloc` rather than `loc` based? Good to know!

Reply:
can you confirm that fh is in the index and not subtracxted from $t_N$ instead?